coderd/prometheusmetrics/aggregator.go

```diff
@@ -24,20 +26,21 @@ const (
 	loggerName = "prometheusmetrics"
 
 	sizeCollectCh = 10
-	sizeUpdateCh  = 1024
+	sizeUpdateCh  = 4096
```

Comment:
Have you considered an unbounded channel? (E.g. a data type that uses channel, but appends backlog to slice). Avoids up-front allocation and can expand infinitely, although, I do think we want a maximum limit for sure.

Reply:
Yeah, scaling indefinitely might be lethal, so I would rather keep the limit :) But I like your advice, I admit that I haven't considered it 🤔

Reply:
Ok, switched to `metricKey`, but I wasn't able to include `[]metricLabel` in it (compiler complained). Is there any better way of handling it?